### DISCUSSION TITLE
Powershell ReportingServicesTools module on Windows image

### DISCUSSION THREAD

**Discussion by ppalma-millenniumbcp** [actions/virtual-environments#6290] [Installed software]
Hi all,

would it be possible to include ReportingServicesTools module (https://github.com/microsoft/ReportingServicesTools) on Windows image?

Thanks in advance!

**Comment by miketimofeev** [comment]
@pauloafp could you please submit a tool request issue?

**Comment by ppalma-millenniumbcp** [comment]
Hi,

I've never submit a tool request issue before, but I'll try and see if I do it correctly!

Thanks!

**Comment by igorboskovic3** [comment]
https://github.com/actions/runner-images/issues/6296
